app/router.js

```diff
@@ -12,8 +12,8 @@ Router.map(function() {
     this.route('survey', function() {
       this.route('results');
       this.route('consent');
-      this.route('complete');
     });
+    this.route('complete', { path: '/survey/complete'});
```

Comment:
When I tried it out in the browser, I got sent to this URL:

`http://localhost:4200/survey/complete`

Instead of the expected url, `/participate/survey/complete`

This is relevant because:

1. There does not appear to be a top-level survey route (so instead of flowing between two related pages in the url bar, the user suddenly appears to jump to a totally different section of the website).

2.The url implies there is a section called `/survey`. When I URL hack (as though I were a user who wanted to to try the survey again), the browser infinitely loads, but without sending any network requests.

Also, there seems to be a failing unit test. Can you please take a look at these items before this PR moves forward?